usaspending_api/references/management/commands/load_rosetta.py

```diff
@@ -40,7 +40,7 @@
 class Command(BaseCommand):
     help = "Loads an Excel spreadsheet of DATA Act/USAspending data names across the various systems into <>"
 
-    s3_public_url = "https://files{}.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx"
+    s3_public_url = settings.DATA_DICTIONARY_DOWNLOAD_URL
```

Comment:
I believe the changes in this file need to be reverted.  The deployment scripts update the `DOWNLOAD_ENV` setting which is used to determine which URL to pull from.  In fact, maybe DATA_DICTIONARY_DOWNLOAD_URL could be updated in the settings file to use the same logic so everywhere it's used it's pointing to the correct location.  That sounded better in my head.  Let me know if what I said didn't make sense and I'll try to esplain more betterer.